geospace-http/src/main/scala/com/socrata/geospace/http/client/CoreServerClient.scala

```diff
@@ -117,7 +117,7 @@ class CoreServerClient(httpClient: HttpClient,
       case Some(rb) =>
         val request = buildRequest(rb)
         logger.info("Request: " + request)
-        for (response <- httpClient.execute(request)) yield {
+        for (response <- httpClient.execute(request)) {
```

Comment:
Does this actually work? I would have thought that removing the yield makes T Unit, which means the post method above can't return Try[JValue]. Am I missing something?